src/store/modules/designs.ts

```diff
@@ -14,6 +14,7 @@ export interface DesignItem {
     gene_knockouts: string[];
     reaction_knockins: string[];
     reaction_knockouts: string[];
+    added_reactions?: Object[];
```

Comment:
Hmm, I think we should overwrite `reaction_knockins` instead of adding a new property here.